orb-ui/src/engine/mod.rs

```diff
@@ -419,6 +419,9 @@ struct Runner<const RING_LED_COUNT: usize, const CENTER_LED_COUNT: usize> {
     operator_signup_phase: operator::SignupPhase,
     sound: sound::Jetson,
     capture_sound: sound::capture::CaptureLoopSound,
+    /// api_mode. When set, update the UI one last time and then pause the engine, see `paused` below.
+    api_mode: bool,
```

Comment:
Rename to `is_api_mode`. I prefer when boolean state is prefixed with `is`

Reply:
done